packages/cli/package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "@dhis2/cli-utils-cypress",
+    "description": "DHIS2 Cypress configuration and utilities",
+    "repository": {
```

Comment:
You can use the last form of the `repository` field [documented here](https://docs.npmjs.com/files/package.json#repository) which allows you to specify a folder within the repo